private_set_intersection/preload.bzl

```diff
@@ -104,3 +105,11 @@ def psi_preload():
             strip_prefix = "bazel-skylib-1.0.2",
             url = "https://github.com/bazelbuild/bazel-skylib/archive/1.0.2.tar.gz",
         )
+
+    if "rules_proto_grpc" not in native.existing_rules():
```

Comment:
Is rules_proto_grpc still needed?

Reply:
We use it for

 load("@rules_proto_grpc//python:defs.bzl", "python_proto_library")

Reply:
Okay, I wasn't sure which python proto implementation we should use, since there seem to be several. But let's stick to this one.